mkdocs.yml

```diff
@@ -30,51 +30,60 @@ theme:
+    - Data Licenses: "user-guide-sharing/data-licenses.md"
+    - Converting Data to NWB:
+      - Overview: "user-guide-sharing/converting-data/index.md"
+      - NWB GUIDE: "user-guide-sharing/converting-data/nwb-guide.md"
+      - NeuroConv: "user-guide-sharing/converting-data/neuroconv.md"
+      - PyNWB and MatNWB: "user-guide-sharing/converting-data/pynwb-matnwb.md"
+    - Validating NWB Files: "user-guide-sharing/validating-files.md"
+    - Uploading Data: "user-guide-sharing/uploading-data.md"
+    - Dandiset Metadata: "user-guide-sharing/dandiset-metadata.md"
+    - Publishing Dandisets: "user-guide-sharing/publishing-dandisets.md"
+    - Citing your own Dandiset: "user-guide-sharing/citing-your-dandiset.md"
+    - Contributing an example notebook: "user-guide-sharing/contributing-notebook.md"
+  - "User Guide: Using Data":
+    - Exploring Dandisets: "user-guide-using/exploring-dandisets.md"
+    - Accessing Data:
+      - Overview: "user-guide-using/accessing-data/index.md"
+      - Downloading: "user-guide-using/accessing-data/downloading.md"
+      - Streaming: "user-guide-using/accessing-data/streaming.md"
+      - External Services: "user-guide-using/accessing-data/external-services.md"
+    - Citing Dandisets: "user-guide-using/citing-dandisets.md"
+    - DANDI Hub: "user-guide-using/dandi-hub.md"
   - Tutorials:
     - Dandiset Examples: https://dandi.github.io/example-notebooks
-    - DANDI User Guide, Part I: "example-notebooks/dandi/DANDI User Guide, Part I.ipynb"
-    - DANDI User Guide, Part II: "example-notebooks/dandi/DANDI User Guide, Part II.ipynb"
-    - Simple Dandiset Search: "example-notebooks/tutorials/neurodatarehack_2024/simple_dandiset_search.ipynb"
-    - Advanced Asset Search: "example-notebooks/tutorials/cosyne_2023/advanced_asset_search.ipynb"
-    - Streaming and interacting with NWB data: "example-notebooks/tutorials/bcm_2024/analysis-demo.ipynb"
-    - NWB Widget Demo: "example-notebooks/demos/NWBWidget-demo.ipynb"
+  - Developer Guide:
+    - System Architecture: "developer-guide/system-architecture.md"
+    - Developer Notes: "developer-guide/developer-notes.md"
+    - Integrate External Services with DANDI: "developer-guide/integrate-external-services.md"
+    - Contributing Documentation: "developer-guide/contributing-documentation.md"
+    - Create DANDI Instance:
+      - Overview: "developer-guide/creating-dandi-instance/index.md"
+      - Initialize Vendor Accounts: "developer-guide/creating-dandi-instance/initialize_vendors.md"
+      - DANDI Authentication: "developer-guide/creating-dandi-instance/dandi_authentication.md"
+      - DANDI Client: "developer-guide/creating-dandi-instance/dandi_cli.md"
+      - DANDI Infrastructure: "developer-guide/creating-dandi-instance/dandi_infrastructure.md"
+      - DANDI Archive: "developer-guide/creating-dandi-instance/dandi_archive.md"
+      - DANDI Hub: "developer-guide/creating-dandi-instance/dandi_hub.md"
```

Comment:
Just for consistency, could you please rename these files to have dashes instead of underscores?

Reply:
done